src/njoy21/input/CCCCR/Card1/Nbrks.hpp

```diff
@@ -0,0 +1,14 @@
+struct Nbrks{
+  using Value_t = int;
```

Comment:
Because zero is an acceptable value and there are further restrictions ( i.e. - nbrks, nisot, and ndlay are all less than zero ), I think rolling our own makes the most sense here.

Reply:
Closer examination of the manual specifies that they can still have either sign, even though they're always binary.  I'll see if I can't simplify the verify functions accordingly.

Reply:
So, argument::common::Nout comes with the unnecessary requirement that the two arguments have the same sign, so I'm not using that.  I have changed these verify functions to ensure both signs can be accepted.

Reply:
Thanks for double checking.